Backend/test/userFacadeTest.ts

```diff
@@ -0,0 +1,156 @@
+var path = require('path')
+require('dotenv').config({ path: path.join(process.cwd(), '.env') })
+import { expect } from "chai";
+import { ApiError } from '../src/customErrors/apiError';
+import bcrypt from "bcryptjs";
+import UserDataAccessorObject from "../src/dataAccessorObjects/userDAO";
+import IUser from "../src/interfaces/IUser";
+import UserFacade from "../src/facades/userFacade";
+
+const schema: string = process.env.DATABASE_SCHEMA_TEST || "";
+let UDAO: UserDataAccessorObject;
+let facade: UserFacade;
+
+describe("Verify the UserFacade", () => {
```

Comment:
I've compeltely forgotten how tests work in JavaScript. Forgot it was even a thing. So these are Chai tests, correct? 
How do you run them?

Reply:
Those are tests created with the library `Chai`, yes. We use `Mocha` to run the tests. I added the following script to `package.json`:
`"test": "mocha --exit -r ts-node/register test/**/*.ts"`